upup/pkg/fi/cloudup/awstasks/autoscalinggroup.go

```diff
@@ -85,6 +87,8 @@ type AutoscalingGroup struct {
 	SuspendProcesses *[]string
 	// Tags is a collection of keypairs to apply to the node on launch
 	Tags map[string]string
+	// TargetGroupARNs is a list of ALB/NLB target group ARNs to add to the autoscaling group
+	TargetGroupARNs []*TargetGroup
```

Comment:
nit, since it is a list of targetgroup tasks rather than just their ARNs
```suggestion
	// TargetGroups is a list of ALB/NLB target groups to add to the autoscaling group
	TargetGroups []*TargetGroup
```